open_discussions/settings.py

```diff
@@ -303,90 +381,41 @@
     # Update the user record with any changed info from the auth service.
     "social_core.pipeline.user.user_details",
     # Resolve outstanding channel invitations
-    "authentication.pipeline.invite.resolve_outstanding_channel_invites",
+    # "authentication.pipeline.invite.resolve_outstanding_channel_invites",
     # Create the moira list associations for the user if any.
     "authentication.pipeline.user.update_moira_lists",
     # update the user's managed channels
-    "authentication.pipeline.user.update_managed_channel_memberships",
+    # "authentication.pipeline.user.update_managed_channel_memberships",
```

Comment:
I think we should leave these in place for now, is there a reason you were removing them?

Reply:
Oh shoot, nope that was me messing around - thank you